Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Applications/Browser/CachedElementTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Aquality.Selenium.Core.Elements;
+using Aquality.Selenium.Core.Elements.Interfaces;
+using Aquality.Selenium.Core.Tests.Applications.WindowsApp.Elements;
+using Aquality.Selenium.Core.Waitings;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+
+namespace Aquality.Selenium.Core.Tests.Applications.Browser
+{
+    public class CachedElementTests : TestWithBrowser
+    {
+        private static readonly By RemoveButtonLoc = By.XPath("//button[.='Remove']");
+        private static readonly By ContentLoc = By.Id("checkbox");
+        private static readonly Uri DynamicContentUrl = new Uri("http://the-internet.herokuapp.com/dynamic_controls");
```

Comment:
if this url is used in several places probably better to have some static class with a list of such URLs

Reply:
fixed